src/carma-streets/ParseTimeSyncToCSV.py

```diff
@@ -0,0 +1,34 @@
+from csv import writer
+from pathlib import Path
+from enum import Enum
+import argparse
+import ParseKafkaLog
+from KafkaLogMessage import KafkaLogMessage, KafkaLogMessageType
+
+
+def parse_timesync_log(inputfile: Path, outputfile: Path):
```

Comment:
This function can be renamed to something more descriptive. Perhaps `extract_time_sync_msgs_to_csv()`.

Reply:
responded see comment above about parser definition